.cloudbuild/README.md

```diff
@@ -28,20 +28,40 @@ gcloud config set project keras-team-test
 To add/update dependency for GPU tests:
 - Add/update dependency to `requirements.txt`
 - Create a `Dockerfile` with the following contents:
-
   ```
-  FROM tensorflow/tensorflow:2.11.0-gpu
+  FROM tensorflow/tensorflow:2.12.0-gpu
   RUN apt-get -y update
   RUN apt-get -y install git
   RUN git clone https://github.com/keras-team/keras-nlp.git
   RUN cd keras-nlp
   RUN pip install -r keras-nlp/requirements.txt
   ```
-
 - Run the following command from the directory with your `Dockerfile`:
-
   ```
-  gcloud builds submit --region=us-west1 --tag us-west1-docker.pkg.dev/keras-team-test/keras-nlp-test/keras-nlp-image:deps --timeout=30m
+  gcloud builds submit --region=us-west1 --tag us-west1-docker.pkg.dev/keras-team-test/keras-nlp-test/keras-nlp-image-tensorflow:deps --timeout=30m
+  ```
+- Repeat the last two steps for Jax and Torch (replacing "tensorflow" with "jax"
+ or "torch" in the docker image target name). `Dockerfile` for jax:
+  ```
+  FROM nvidia/cuda:11.7.1-base-ubuntu20.04
```

Comment:
why not `11.8.0` here? that's the version tf depends on, and it would be nice to consolidate on one version of cuda for our testing. https://www.tensorflow.org/install/pip